x-pack/plugins/cases/public/components/templates/form_fields.test.tsx

```diff
@@ -395,4 +396,47 @@ describe('form fields', () => {
       );
     });
   });
+
+  it('does not render duplicate template tags', async () => {
+    const newProps = {
+      ...defaultProps,
+      currentConfiguration: {
+        ...defaultProps.currentConfiguration,
+        templates: [
+          {
+            key: 'test_template_1',
+            name: 'Test',
+            tags: ['one', 'two'],
+            caseFields: {},
+          },
+          {
+            key: 'test_template_2',
+            name: 'Test 2',
+            tags: ['one', 'three'],
+            caseFields: {},
+          },
+        ],
+      },
+    };
+
+    appMockRenderer.render(
+      <FormTestComponent onSubmit={onSubmit} formDefaultValue={formDefaultValue}>
+        <FormFields {...newProps} />
+      </FormTestComponent>
+    );
+
+    const caseTags = await screen.findByTestId('template-tags');
+
+    userEvent.click(within(caseTags).getByTestId('comboBoxToggleListButton'));
```

Comment:
```suggestion
    userEvent.click(await within(caseTags).findByTestId('comboBoxToggleListButton'));
```

Reply:
Because we awaited in `await screen.findByTestId('template-tags');` and we are searching within the already found element I thought that the DOM was already there. Wdyt?

Reply:
It should be, it's just a sanity check for the inevitable flaky tests 😛 Up to you, just a nit!